src/flavors/duwamish_flavor/config.yml

```diff
@@ -60,6 +60,16 @@ map:
       url: https://k7b7dyc4v3.execute-api.us-west-2.amazonaws.com/production/mapbox/landmarks?url=http://a.tiles.mapbox.com/v4/smartercleanup.k9dcl2i9/features.json?access_token=pk.eyJ1Ijoic21hcnRlcmNsZWFudXAiLCJhIjoiTnFhUWc2cyJ9.CqPJH-9yspIMudowQJx2Uw
       type: landmark
 
+    - id: duwamish
+      url: http://dev-api.heyduwamish.org/api/v2/smartercleanup/datasets/duwamish
```

Comment:
I am thinking that we should move the url's into environment variables, for a couple reasons:

 * It will remove the need for query params, thus cleaning up the code (more info on removing the query params is below)
 * It is deceiving to have our Shareabouts url in the config, when it really needs to be coupled with a dataset key to function properly. So I think configuring both url and dataset key in the .env file makes the most sense